src/Interactive/EditorFeatures/VisualBasic/BasicInteractiveEditorFeatures.vbproj

```diff
@@ -153,6 +154,9 @@
     <PublicAPI Include="PublicAPI.Shipped.txt" />
     <PublicAPI Include="PublicAPI.Unshipped.txt" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Interactive\CompletionProviders\ReplCommandCompletionProvider.vb" />
```

Comment:
I assume you didn't add this manually, but it's pretty weird that it's not with the other '<Compile>' elements.